docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -14,35 +14,34 @@ ms.assetid: 8db75596-19a9-4eda-b20d-a12d517c8cc1
 
 # Getting started with F# in Visual Studio
 
-F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs), if you haven't already.  This article uses the Visual Studio 2015 Community Edition, but you can use F# with the version of your choice.
+F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs), if you haven't already.  This article uses the Visual Studio 2017 Community Edition, but you can use F# with the version of your choice.
 
-## Installing the Visual F# tools
+## Installing F# #
 
-Visual Studio will first initialize the installer.  After it is intilized, select **Custom** as shown here:
+If you're downloading Visual Studio for the first time, it will install the installer.  Install any version of Visual Studio 2017 from the installer.  If you already have it installed, click **Modify**.
 
-![Select Custom install](./media/getting-started-vs/vs2015-install-1.png)
+You'll next see a list of Workloads.  You can install F# through any of the following workloads:
 
-Select the Visual F# Tools under Programming Languages here:
+|Workload|Action|
+|--------|------|
+|.NET desktop development|Select **F# Tools** from right-hand side|
+|ASP.NET and web Development|Select **F# Tools** from right-hand side|
+|Data storage and processing|Select **F# Tools** from right-hand side|
+|Mobile development with .NET|No action - F# is installed by default|
 
-![Visual F#](./media/getting-started-vs/vs2015-install-2.png)
+Alernatively, you can click on **Individual Components** and install **F# Tools** from there.  Feel free to select as many Workloads or Individual Components as you like.
```

Comment:
typo: Alernatively

also, according to style guide, the right verb for UI interaction would be select instead of click (I have a hard time with this one but it's [here](https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=26472))